TECH-LEADS-CHECKLIST.md

```diff
@@ -28,8 +28,8 @@ More details on [Unit Testing](automated-testing/unit-testing/readme.md)
 ## CI/CD
 
 - [ ] Project runs CI with automated build and test on each PR.
-- [ ] Project uses CD to manage deployments to a replica environment before PRs are merged to master.
-- [ ] Master is always shippable.
+- [ ] Project uses CD to manage deployments to a replica environment before PRs are merged.
+- [ ] Main branch is always shippable.
```

Comment:
Should this be Master and Develop (if applicable) are always shippable

Reply:
Will take your suggestion of documenting what a main branch is and linking to it.